src/components/PeoplePage.tsx

```diff
@@ -1,32 +1,77 @@
 import { PeopleFilters } from './PeopleFilters';
 import { Loader } from './Loader';
 import { PeopleTable } from './PeopleTable';
+import { useEffect, useState } from 'react';
+import { getPeople } from '../api';
+import { Person } from '../types';
+import { useSearchParams } from 'react-router-dom';
+import { getFilteredPeople } from '../utils/getFiltredPeople';
 
 export const PeoplePage = () => {
-  return (
-    <>
-      <h1 className="title">People Page</h1>
+  const [people, setPeople] = useState<Person[]>([]);
+  const [error, setError] = useState(false);
+  const [loadPeople, setLoadPeople] = useState(false);
+  const [searchParams] = useSearchParams();
 
-      <div className="block">
-        <div className="columns is-desktop is-flex-direction-row-reverse">
-          <div className="column is-7-tablet is-narrow-desktop">
-            <PeopleFilters />
-          </div>
+  const hasError = !loadPeople && error;
+  const noPeople = people.length === 0 && !loadPeople;
```

Comment:
The `noPeople` condition checks if there are no people and loading is complete. However, it might also trigger if there is an error fetching data. Consider updating this condition to ensure it only shows the 'no people' message when there is no error and the loading is complete.